setup.py

```diff
@@ -16,7 +16,10 @@
     author="Jeff Irion",
     author_email="jefflirion@users.noreply.github.com",
     packages=["androidtv", "androidtv.adb_manager", "androidtv.basetv", "androidtv.androidtv", "androidtv.firetv"],
-    install_requires=["adb-shell>=0.4.0", "pure-python-adb>=0.3.0.dev0"],
+    install_requires=[
+        "adb-shell>=0.4.0",
+        "pure-python-adb @ git+https://github.com/CloCkWeRX/pure-python-adb@v0.3.0.pre",  # TODO: Change to https://github.com/spm5065/pure-python-adb when a tag is available
```

Comment:
```suggestion
        "pure-python-adb @ git+https://github.com/CloCkWeRX/pure-python-adb@v0.3.0.pre",  # This will change to https://github.com/spm5065/pure-python-adb when a tag is available
```